packages/commonwealth/client/scripts/controllers/app/webWallets/metamask_web_wallet.ts

```diff
@@ -42,6 +43,17 @@ class MetamaskWebWalletController implements IWebWallet<string> {
     return this._accounts || [];
   }
 
+  public async getRecentBlock(): Promise<BlockInfo> {
+    // await this.enable()
```

Comment:
What does this do? Why is it commented out?

Reply:
This was originally there to ensure Metamask was initialized before we tried to fetch a block, but since Alex's refactor I don't think it's necessary now.